analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -535,18 +556,27 @@ private static void VerifyClassMainVB(string members)
 $@"Public Class Sample
     {members}
 End Class";
-        VerifyCode<TestSERunnerVB>(code, ProjectType.Product, ImmutableArray<ParseOptions>.Empty, null, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
+        VerifyCode<TestSERunnerVB>(code, ProjectType.Product, ImmutableArray<ParseOptions>.Empty, null, false, MainScopeAssignmentRuleCheck.SMain, BinaryRuleCheck.SBinary);
     }
 
-    private static void VerifyCode<TRunner>(string code, ProjectType projectType, ImmutableArray<ParseOptions> parseOptions, string sonarProjectConfigPath, params DiagnosticDescriptor[] onlyRules)
-        where TRunner : SymbolicExecutionRunnerBase, new() =>
-        new VerifierBuilder<TRunner>()
-            .AddReferences(TestHelper.ProjectTypeReference(projectType))
-            .AddSnippet(code)
-            .WithSonarProjectConfigPath(sonarProjectConfigPath)
-            .WithOptions(parseOptions)
-            .WithOnlyDiagnostics(onlyRules)
-            .Verify();
+    private static void VerifyCode<TRunner>(string code, ProjectType projectType, ImmutableArray<ParseOptions> parseOptions, string sonarProjectConfigPath, bool inTopLevelStatements, params DiagnosticDescriptor[] onlyRules)
+        where TRunner : SymbolicExecutionRunnerBase, new()
+    {
+        var verifier = new VerifierBuilder<TRunner>()
+                 .AddReferences(TestHelper.ProjectTypeReference(projectType))
+                 .AddSnippet(code)
+                 .WithSonarProjectConfigPath(sonarProjectConfigPath)
+                 .WithOptions(parseOptions)
+                 .WithOnlyDiagnostics(onlyRules);
```

Comment:
You can also turn off concurrent analysis here for everyone. It doesn't make sense to test it